test/test_data_collector.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ros_cross_compile.data_collector import DataCollector
+from ros_cross_compile.data_collector import DataWriter
+from ros_cross_compile.data_collector import Datum
+
+
+def test_datum_construction():
+    test_datum = Datum('test_stat', 3, 'tests', 130.222, True)
+    assert test_datum
+
+
+def test_collector_construction():
+    test_collector = DataCollector()
+    assert test_collector
+
+
+def test_data_collection():
+    test_collector = DataCollector()
+
+    test_datum_a = Datum('test_stat_1', 3, 'tests', 130.452, True)
```

Comment:
Consider moving these values to named variables so they can be reusable/explicit.